projects/coredns/coredns/Makefile

```diff
@@ -57,6 +58,14 @@ images: binaries
 		--local dockerfile=./docker/linux \
 		--local context=. \
 		--output type=image,oci-mediatypes=true,name=$(IMAGE),push=true
+	buildctl \
+		build \
+		--frontend dockerfile.v0 \
+		--opt platform=linux/amd64 \
```

Comment:
This is only building the amd64 whereas the one above is also building arm64. Is this is intentional?

Reply:
/hold
Good call. This PR is actually a work in progress. Wanted to make a few more changes like adding checksums and placing artifacts in the appropriate directory since we will probably be releasing this. I'll add ARM64 builds along with that change.